thirdparty/dropbear/dropbear-2018.76-kindle-pubkey-gen-hack.patch

```diff
@@ -0,0 +1,28 @@
+diff -ur dropbear-2018.76/gensignkey.c dropbear-2018.76-PATCHED/gensignkey.c
+--- dropbear-2018.76/gensignkey.c	2018-02-27 15:25:10.000000000 +0100
++++ dropbear-2018.76-PATCHED/gensignkey.c	2018-04-14 17:17:03.419481762 +0200
+@@ -139,17 +139,14 @@
+ 	if (ret == DROPBEAR_FAILURE) {
+ 		goto out;
+ 	}
++   /* Hard links are not possible and renam. */
```

Comment:
This refers to FAT32?

Reply:
yes I think, the patch is comming from here https://github.com/termux/termux-packages/blob/00796411c0de50cad2705a0ac714b47d421f9c9f/packages/dropbear/gensignkey.c.patch